macros/SpecData.json

```diff
@@ -429,6 +429,11 @@
     "url": "https://drafts.csswg.org/css-grid/",
     "status": "CR"
   },
+  "CSS Grid2": {
```

Comment:
Nit: looking at the other keys, this should be "CSS Grid 2" (with space) or "CSS Grid Level 2".